apm-network/src/main/proto/TraceSegmentService.proto

```diff
@@ -22,11 +22,10 @@ message UniqueId {
 
 message TraceSegmentObject {
     UniqueId traceSegmentId = 1;
-    repeated TraceSegmentReference refs = 2;
-    repeated SpanObject spans = 3;
-    int32 applicationId = 4;
-    int32 applicationInstanceId = 5;
-    bool isSizeLimited = 6;
+    repeated SpanObject spans = 2;
+    int32 applicationId = 3;
+    int32 applicationInstanceId = 4;
+    bool isSizeLimited = 5;
```

Comment:
Is this what it means the number of span more than the maximum number? 
I can't imagine how collector and UI module to do when isSizeLimited be true.

Reply:
@peng-yongsheng `isSizeLimited` is for memory protection in agent, when there are too many spans in single one segment. It should not be used in UI, to tell user what happened in agent.